lib/constants/app_controllers.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:flutter/cupertino.dart';
+
+class AppControllers {
+  static TextEditingController searchController = TextEditingController();
+}
```

Comment:
controllerをグローバルな変数として定義すると関係ないクラスからもいじれてしまうので、StatefulWidgetまたはNotifier内で定義した方がいいかも

Reply:
修正しました！ [fae9395](https://github.com/shinonome-inc/pg_mobile/pull/12/commits/fae939593e9100aef2e40ff6c046abfd92ec5383)

Reply:
しっかり修正できています👍
本実装ではdisposeも意識してみましょう！

> Remember to [dispose](https://api.flutter.dev/flutter/foundation/ChangeNotifier/dispose.html) of the [TextEditingController](https://api.flutter.dev/flutter/widgets/TextEditingController-class.html) when it is no longer needed. This will ensure we discard any resources used by the object.

引用：https://api.flutter.dev/flutter/widgets/TextEditingController-class.html